config/src/main/java/org/springframework/security/config/annotation/web/configurers/AuthorizeHttpRequestsConfigurer.java

```diff
@@ -373,6 +375,32 @@ public AuthorizationManagerRequestMatcherRegistry anonymous() {
 			return access(AuthenticatedAuthorizationManager.anonymous());
 		}
 
+		public class AuthorizedUrlVariable {
```

Comment:
Please add a JavaDoc for this class. Please feel free to add an `@author` tag for yourself.